definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -7014,5 +7014,47 @@
     "CaseEventID": "createGeneralOrder",
     "UserRole": "citizen",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "NO_FAULT_DIVORCE",
+    "CaseEventID": "draftCreate",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "NO_FAULT_DIVORCE",
+    "CaseEventID": "draftCreate",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "NO_FAULT_DIVORCE",
+    "CaseEventID": "draftCreate",
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "NO_FAULT_DIVORCE",
+    "CaseEventID": "draftCreate",
+    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
Unsure if you want all these users to be able to see the draft state - might just citizens to have CRU and superusers to have R?

Reply:
fair, wasnt sure which user had which authentication hence i gave all of them R except for citizen.